src/main/java/jenkins/plugins/logstash/LogstashBuildWrapper.java

```diff
@@ -46,17 +51,24 @@
 
 /**
  * Build wrapper that decorates the build's logger to insert a
- * {@link LogstashNote} on each output line.
+ * Logstash note on each output line.
  *
  * @author K Jonathan Harker
  */
 public class LogstashBuildWrapper extends BuildWrapper {
 
+  @CheckForNull
+  private SecureGroovyScript secureGroovyScript;
+
   /**
    * Create a new {@link LogstashBuildWrapper}.
    */
   @DataBoundConstructor
-  public LogstashBuildWrapper() {}
+  public LogstashBuildWrapper(
+      @CheckForNull SecureGroovyScript secureGroovyScript
```

Comment:
Breaks the binary compatibility. The default constructor should be retained

Reply:
I'm very new to plugin dev task. My gut feeling told me that given jelly is changed, the default constructor will never be called by Jenkins thus retaining a parameterless constructor would be useless.
If you want me to keep one I'm fine with it. Just tell me whether it should be annotated as `@DataBoundConstructor` or not. Other than that, it would be an easy change.